### PR TITLE
Change title back to project name after submitting the form

### LINKED ISSUE #130: After having submited the form, the title doesn't change back to the homepage.
When you open a form you have the title of the form, it doesn't change when you go back to the home page.

### PR DESCRIPTION
**Pull request by sahildua2305**
Fixes #130

/cc @almet @Natim

### CODE REVIEW

File: formbuilder/components/RecordCreated.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React from "react";
-
+import config from "../config";
 
 export default function RecordCreated(props) {
```
Comment (Natim) [inline formbuilder/components/RecordCreated.js]: I think we want to do that when the Homepage loads.

File: formbuilder/components/RecordCreated.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+
+	// Issue #130 - Change title back to project name after submitting the form
+	document.title = config.projectName;
```
Comment (Natim) [inline formbuilder/components/RecordCreated.js]: To many spaces before `document`. It should be aligned with return

File: formbuilder/components/RecordCreated.js (hunk 1 above)
Comment (almet) [inline formbuilder/components/RecordCreated.js]: When the homepage loads, the title is already the correct one :)

**Comment by almet** [comment]
Thanks, just a few nitpicks but we should be good to merge soon.

File: formbuilder/components/RecordCreated.js (hunk 2 above)
Comment (sahildua2305) [inline formbuilder/components/RecordCreated.js]: Fixed!

**Comment by sahildua2305** [comment]
Pushed the changes @almet 👍

**Comment by almet** [comment]
It's now merged, thanks :-)

**Pull request merged**